appeals/web/src/server/appeals/appeal-details/representations/lpa-statement/redact/redact.controller.js

```diff
@@ -36,9 +36,21 @@ export async function renderConfirm(request, response) {
  * */
 export function postRedact(request, response) {
 	const {
-		params: { appealId }
+		params: { appealId },
+		currentAppeal,
+		session
 	} = request;
 
+	if (
+		!currentAppeal.allocationDetails?.level ||
+		!currentAppeal.allocationDetails?.specialisms?.length
+	) {
+		session.acceptLPAStatement.forcedAllocation = true;
```

Comment:
This param controls the back button on the levels page and whether or not to show the yes/no question on the CYA pages